components/design-dropdown/design-dropdown.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Icon, MenuToggle } from "@patternfly/react-core";
+import PenToolIcon from "../Navigation/pen-tool";
+
+const DesignDropdown = ({
+  width,
+  height,
+}: {
+  width?: string | number;
+  height?: string | number;
```

Comment:
I don't see these props used in this component.

Reply:
@Hyperkid123 @karelhala has me all all these because the tests were failing `<PenToolIcon width="22" height="22" />`